setup.py

```diff
@@ -21,6 +21,7 @@
       license='MIT',
 
       packages=['lmchallenge', 'lmchallenge.core'],
+      include_package_data=True,
```

Comment:
this is required to include files listed in `Manifest.in` into the package and install them when the package is installed.
see https://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files